gcloud/bigquery/client.py

```diff
@@ -102,20 +102,18 @@ def dataset(self, dataset_name):
         """
         return Dataset(dataset_name, client=self)
 
-    def _job_from_resource(self, resource):
+    def job_from_resource(self, resource):
```

Comment:
Hmm?  Using private methods from classes in another module would set of lint, for one.